.travis.yml

```diff
@@ -31,7 +31,7 @@ services:
 cache:
   ccache: true
   directories:
-    - $BASE_BUILD_DIR/ci/scratch/.ccache
```

Comment:
is this dropping cache?

Reply:
we don't use Travis

Reply:
I guess that is true :D

Reply:
We don't use Travis, I'm just keeping this changes for backport compatibility. It doesn't affect any caches bcs no travis at all. But if there's any good idea, we can adapt it for our CI.

Reply:
imo it's fine to just backport changes for Travis as is and implement similar dashified ones for Gitlab

what is missing in our backport is release cache they added when they changed this line, we could add one too - pls see (and test) 7fe8c49722

this PR:
first run https://gitlab.com/dashpay/dash/-/jobs/4911762972#L106 (downloading)
second run https://gitlab.com/dashpay/dash/-/jobs/4934280594#L106 (downloading)

7fe8c49722:
first run https://gitlab.com/UdjinM6/dash/-/jobs/4935068647#L59 (downloading)
another branch with a dummy commit on top https://gitlab.com/UdjinM6/dash/-/jobs/4935596048#L62 (using cache)